README.md

```diff
@@ -16,6 +16,19 @@ definition 3D LIDARs`<sup>3</sup>.
 The current ``master`` branch works with ROS Kinetic and Melodic.
 CI builds are currently run for Kinetic and Melodic.
 
+# Windows
+
+To build for ROS on Windows, start by installing ROS1 following the instructions at [Microsoft's ROS landing page](http://aka.ms/ros).
+
+After installing, launching a ROS terminal, install the pcap dependency using the Microsoft vcpkg package manager:
```

Comment:
[nit] "After installing, launch a ROS terminal as described in the instructions. In the terminal, install the pcap..."